internal/backend/backend.go

```diff
@@ -63,7 +61,7 @@ func (b *Backend) AddUser(ctx context.Context, userID string, conn connector.Con
 		return err
 	}
 
-	db, err := b.newDB(userID)
+	db, err := db.NewDB(b.dir, userID)
```

Comment:
`db.NewDB` appends `/db` to the given directory before using it. Probably better to do that here (like we do on line 59 with `/store`) and keep the db package simple, thoughts?

Reply:
Good call. Fixed.